internal/analytics/client/sync/reporters/ga-state.go

```diff
@@ -91,5 +92,7 @@ func legacyDimensionMap(d *dimensions.Values) map[string]string {
 		"17": p.PStr(d.InstanceID),
 		"18": p.PStr(d.Headless),
 		"19": p.PStr(d.CommitID),
+		"20": p.PStr(d.Command),
+		"21": strconv.Itoa(p.PInt(d.Sequence)),
```

Comment:
Not sure how much it matters for GA reporting but isn't there a limit to the number of dimensions we can have? Not sure what the limit is either I just seem to recall this being brought up at some point.

Reply:
We don't report to GA anymore :) We'll probably refactor this out in the future.